.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: rosdistro-ci
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python: ['3.7', '3.8', '3.9', '3.10']
+        include:
+        - os: ubuntu-18.04
+          python: '2.7'
+        - os: ubuntu-18.04
+          python: '3.6'
+    name: rosdistro tests
+    runs-on: ${{matrix.os}}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{matrix.python}}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{matrix.python}}
+    - name: Install dependencies
+      run: |
+        python -m pip install -U -e .[test] pytest-cov -c constraints.txt
+    - name: Run tests
+      run: |
+        python -m pytest --cov test
```

Comment:
Probably not critical but the coverage reported by the change is different than the one in travis and probably incomplete:
```
2022-03-30T21:51:04.3364048Z 
2022-03-30T21:51:04.3364479Z ---------- coverage: platform linux, python 3.7.12-final-0 -----------
2022-03-30T21:51:04.3365309Z Name                                    Stmts   Miss  Cover
2022-03-30T21:51:04.3366070Z -----------------------------------------------------------
2022-03-30T21:51:04.3366677Z test/__init__.py                            3      0   100%
2022-03-30T21:51:04.3367185Z test/test_cache.py                          7      0   100%
2022-03-30T21:51:04.3368232Z test/test_distribution.py                  29      0   100%
2022-03-30T21:51:04.3368890Z test/test_doc.py                           27      0   100%
2022-03-30T21:51:04.3369388Z test/test_doc_build.py                     18      0   100%
2022-03-30T21:51:04.3369991Z test/test_index.py                         69      3    96%
2022-03-30T21:51:04.3370592Z test/test_manifest_providers.py            85      2    98%
2022-03-30T21:51:04.3371109Z test/test_release.py                       27      0   100%
2022-03-30T21:51:04.3371737Z test/test_release_build.py                 43      0   100%
2022-03-30T21:51:04.3372382Z test/test_repository_specification.py      17      0   100%
2022-03-30T21:51:04.3372993Z test/test_source.py                        27      0   100%
2022-03-30T21:51:04.3373487Z test/test_source_build.py                  18      0   100%
2022-03-30T21:51:04.3374109Z test/test_verify.py                         9      0   100%
2022-03-30T21:51:04.3374704Z test/test_writer.py                        31     17    45%
2022-03-30T21:51:04.3375402Z -----------------------------------------------------------
2022-03-30T21:51:04.3375999Z TOTAL                                     410     22    95%
2022-03-30T21:51:04.3376299Z 
```
Not a pytest expert but could be something like:
```suggestion
        python -m pytest --cov=src/rosdistro test
```

Reply:
[This configuration](https://github.com/ros-infrastructure/rosdistro/blob/8f15df1779fc54b2dd7f0d5719c1e1a4f43d5771/setup.cfg#L12-L13) should be taking care of this. I'll investigate and try to figure out what's going on.

Reply:
Well at least it was a straightforward fix. It seems pytest was considering the `test` argument on the command line to be an argument for `--cov`, not a positional as it is intended. So it was overriding the value in `setup.cfg`.

I swapped the order of the arguments. Let's see if that does the trick.

Nice catch!

Reply:
> [This configuration](https://github.com/ros-infrastructure/rosdistro/blob/8f15df1779fc54b2dd7f0d5719c1e1a4f43d5771/setup.cfg#L12-L13) should be taking care of this. I'll investigate and try to figure out what's going on.

aah ok!

> I swapped the order of the arguments. Let's see if that does the trick.

It did!